### PR TITLE
Docs: Create add-a-panel.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
This is a draft because many of the dependency docs that need to be linked have either not been created or not been merged yet.

Images are in https://github.com/grafana/website/pull/1467

This is the task topic for adding a panel to a dashboard. The panel editor, queries, and visualizations are all documented in different topics.

### CODE REVIEW

File: docs/sources/panels/add-a-panel.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## 2. Edit panel settings
+
+While not required, we recommend that you add a helpful title and description to your panel. You can use variables you have defined in either field, but not global variables.
+
+![](/img/docs/panels/panel-settings-7-0.png)
+
+**Panel title -** Text entered in this field is displayed at the top of your panel in the Panel editor and in the dashboard.
+
+**Description -** Write a description of the panel and the data you are displaying. Pretend you are explaining it to a new user six months from now, when it is no longer fresh in your mind. Future editors (possibly yourself) will thank you.
```
Comment (torkelo) [inline docs/sources/panels/add-a-panel.md:34]: This is optional and shown in a tooltip in upper left corner (an info icon is shown after adding a description). You can use markdown in the description with links.

**Review by torkelo — APPROVED** [review]
Looks great at to the point!

File: docs/sources/panels/add-a-panel.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   
+     ![](/img/docs/panels/add-panel-icon-7-0.png)
+
+1. Click **Add new panel**.
+
+Grafana creates an empty graph panel with your default data source selected.
+
+## 2. Edit panel settings
+
+While not required, we recommend that you add a helpful title and description to your panel. You can use variables you have defined in either field, but not global variables.
```
Comment (dprokop) [inline docs/sources/panels/add-a-panel.md:28]: thi
```suggestion
While not required, we recommend that you add a helpful title and description to your panel. You can use template variables you have defined in your dashboard.
```

File: docs/sources/panels/add-a-panel.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+## 3. Write a query
+
+Each panel needs at least one query to display a visualization. You write queries in the Query tab of the Panel editor. For more information about the Query tab, refer to [Queries]({{< relref "queries.md" >}}).
+
+1. Choose a data source. In the first line of the Query tab, click the drop-down list to see all available data sources. This list includes all data sources you added. Refer to [Add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}) if you need instructions.
+1. Write or construct a query in the query language of your data source. Options will vary. Refer to your specific [data source documentation]({{< relref "../features/datasources/_index.md" >}}) for specific guidelines.
+
+## 4. Choose a visualization type
+
+In the Visualization section, click a visualization type. Grafana displays a preview of your query with that visualization applied.
```
Comment (dprokop) [inline docs/sources/panels/add-a-panel.md:45]:
```suggestion
In the Visualization section of the Panel editor options pane, click a visualization type. Grafana displays a preview of your query results with that visualization applied. 
```

**Review by dprokop — APPROVED** [review]
Some small nits, other than that 👍

File: docs/sources/panels/add-a-panel.md (hunk 2 above)
Comment (oddlittlebird) [reply docs/sources/panels/add-a-panel.md:28]: We don't actually use the term "template variables" anywhere in the UI, and while people like you and Daniel seem to know what they are, I am not confident that anyone else does. The global variables are listed and defined in the Global variables topic. I will add some clarifying links.

"Template" as defined to me (and in the docs) is any text that includes a variable. So "template variable" is not at all clear.

**Pull request merged**